llmware/library.py

```diff
@@ -782,7 +1245,27 @@ def run_ocr_on_images(self, add_to_library=False,chunk_size=400,min_size=10, rea
 
         """ Convenience method in Library class to pass Library to Parser to run OCR on all of the images
```

Comment:
Delete space before you start the doc.
```suggestion
        """Convenience method in Library class to pass Library to Parser to run OCR on all of the images
```